docs/development.rst

```diff
@@ -83,16 +83,7 @@ check the rendering.
 Javascript package management
 =============================
 
-Use `yarn` for package management.
-
-To upgrade a Javascript package to a newer version use the following. This
-keeps `package.json` and `yarn.lock` in sync as BOTH will be modified:
-
-.. code-block:: shell
-
-   # ATTENTION: Select only those with reasonable version numbers.
-   # Be very careful if the MAJOR number would change
-   yarn upgrade-interactive --latest
+Use `npm` for package management.
```

Comment:
I've removed most of this section as npm has no interactive upgrade mechanism as far as I'm aware.